go/stellar/stellarsvc/frontend.go

```diff
@@ -1282,6 +1288,7 @@ func (s *Server) SendPaymentLocal(ctx context.Context, arg stellar1.SendPaymentL
 	return stellar1.SendPaymentResLocal{
 		KbTxID:  sendRes.KbTxID,
 		Pending: sendRes.Pending,
+		TxID:    sendRes.TxID,
```

Comment:
@buoyad do you want this?

Reply:
I can remove this. I needed it for debugging one time while working on tests for this PR.

Reply:
If you want to keep it, how about returning a PaymentID?

Reply:
Frontend only uses PaymentID

Reply:
Ok, @zapu can you either remove or change to PaymentID?  thanks.

Reply:
will do, thanks!